<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pickmenu</class>
 <widget class="QWidget" name="pickmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l_boss">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выбор древнего</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_chara">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выбор персонажа</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_boss">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>211</width>
     <height>192</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>this one is empty yet</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="list_chara">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>221</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QListWidget" name="info_chara">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>80</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nickname_input">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
